frontend/src/components/NavbarStyled.tsx

```diff
@@ -0,0 +1,14 @@
+import { FontAwesomeIconProps } from "@fortawesome/react-fontawesome";
+
+export interface buttonStates {
```

Comment:
It's good practice to have your interfaces start with capitals; so `ButtonStates`